src/features/components/templates/footer/link-item/index.tsx

```diff
@@ -18,7 +20,8 @@ const FooterLink = ({ item: { data } }: FooterLinkProps) => {
 
     return (
         <Link url={visible_url} size="small" pb="6x">
-            <Localize translate_text={data.text} />
+            {data?.text ? <Localize translate_text={data.text} /> : null}
+            {data?.src ? <Image src={data.src} className={footer_image} /> : null}
```

Comment:
conditional rendering is bad approach, let's create new template like `LinkImage`